internal/journal/voljournal.go

```diff
@@ -635,6 +641,11 @@ func (conn *Connection) ReserveName(ctx context.Context,
 		omapValues[cj.cephSnapSourceKey] = parentName
 	}
 
+	// Update backing snapshot ID for shallow CephFS volume
```

Comment:
I decided to go with "snapshot-backed volumes" instead of "shallow", because it seems a bit more descriptive -- and implies that it's in fact read-only (because cephfs snapshots are read-only). I'll change this and other occurrences of "shallow" to be consistent with this naming.